### DISCUSSION TITLE
Right Hand Drive availability

### DISCUSSION THREAD

**Discussion by Kiruthiharan** [microsoft/AirSim#3345] [Support Q&A]
Hi, currently the simulator is made for a left hand drive car, is there any option to change it to a right hand drive car? , or is it not possible to do so?. Any workarounds that I can use to get this done will be really helpful

**Comment by ahmed-elsaharti** [comment]
#3311 might help with this. You'll need to either create a car pawn with the mesh u want OR modify the existing one like shown in the issue's replies. Then you'll want to change the `"DefaultCar"` [PawnPath](https://microsoft.github.io/AirSim/settings/#pawnpaths) in settings.json

**Reply by Kiruthiharan** [reply]
Thank you, ill check them out
